File: src/main/java/maxipool/getcandleshistoricalbatch/GetCandlesHistoricalBatchApplication.java
package maxipool.getcandleshistoricalbatch;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.cloud.openfeign.EnableFeignClients;

@SpringBootApplication
@ConfigurationPropertiesScan
@EnableFeignClients
public class GetCandlesHistoricalBatchApplication {

	public static void main(String[] args) {
		SpringApplication.run(GetCandlesHistoricalBatchApplication.class, args);
	}

}

File: src/main/kotlin/com/markets/getcandleshistoricalbatch/GetCandlesHistoricalBatchApplication.kt
package com.markets.getcandleshistoricalbatch

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients

@SpringBootApplication
@ConfigurationPropertiesScan
@EnableFeignClients
class GetCandlesHistoricalBatchApplication

fun main(args: Array<String>) {
  runApplication<GetCandlesHistoricalBatchApplication>(*args)
}
